src/core.rs

```diff
@@ -118,13 +118,12 @@ impl Core {
     pub fn scroll_to(&mut self, cursor: &Cursor) -> Result<()> {
         info!("Updating cursor position");
 
-        if let Some(view) = self.views.get_mut(&self.current_view) {
+        if let Some(view) = self.get_view_mut() {
             view.update_cursor(cursor);
-            Ok(())
-        } else {
-            error!("View {} not found", &self.current_view);
-            bail!(ErrorKind::UpdateError);
+            return Ok(());
```

Comment:
Ah yes. That was the part [I said I would not change](https://github.com/little-dude/xi-tui/pull/35#discussion_r114869275). My git skills failed me.